NETEZZA.md

```diff
@@ -1,46 +1,24 @@
-  
+   * Download the 7.2 version of the [Netezza Emulator](https://www14.software.ibm.com/webapp/iwm/web/reg/pick.do?source=swg-im-ibmndn&lang=en_US)
      **NOTE:** you will need to register and create an IBM developer network account
      
    * Install VWware 
-   * Verify the Netezza Emulator requirements and install with the installer
+   * Open the downloaded Netezza VM `.ova` file in VMWare
      **NOTE:** The Netezza Emulator typically requires >= 4G of ram
-
-   * Install the JDBC driver. This creates a seperate directory which contains the nzjdbc.jar needed.
-   * Optionally install the Netezza Administrator GUI (makes life a lot easier)
-     **NOTE:** the default passwords for the emulator are nz/nz for the linux host and admin/password for the database connection
+     **NOTE:** the default passwords for the emulator are `nz`/`nz` for the Linux host and `admin`/`password` for the database connection
   
-2. Setup a Development environment  
-   Unfortunately gradlew doesn't play nice with windows for test. 
-   It is therefore more convenient to set up a seperate VWware linux image on the same virtual network as the emulator. 
-   **NOTE:** this is tested with ubuntu 14.04 LTS with git installed.
-
-   * Download and install an ubuntu ISO and create a development VM in the same VMware environment as the Netezza emulator.
-   * Make sure GIT is installed and you have a working internet connection from the VM
-   * Test if the Netezza Emulator is visible from the development vm (example: $ping 192.168.85.128)
-   * Optionally install the linux client tools on the VM.
-
-3. Configure the connection for the DbFit acceptance tests in the file `dbfit-java\TestDbConnectionDbFitNetezza.properties.custom`:
-   * Set the user name in the 'username=' key.
-   * Set the password in the 'password=' key.
-   * Set the hostname and the port number (format is <hostname>:<port>) in the 'service=' key.
-
-4. Create the database NETEZZATEST and acceptance test database objects either by: -
-   * Running the command './gradlew :dbfit-java-netezza:setUpTestDatabase'
-   or
-   * Create the database NETEZZATEST using either the Administrator GUI or nzsql and
-     running `001_add_objects_needed_for_acceptance_test.sql`.
-
-5. Rename the nzjdbc.jar to nzjdbc-7.1.jar and put it in the custom libs
+2. Configure the connection for the DbFit acceptance tests in the file `dbfit-java\TestDbConnectionDbFitNetezza.properties.custom`:
+   * Set the user name in the `username=` key.
+   * Set the password in the `password=` key.
+   * Set the hostname and the port number (format is `<hostname>:<port>`) in the `service=` key.
 
-6. Build dbfit excluding tests (gradlew clean check install -x test)
+3. Create the database NETEZZATEST by running the command `./gradlew :dbfit-java-netezza:setUpTestDatabase`.
 
-7. Run the tests ./gradlew :dbfit-java:netezza:test
+4. Run the Netezza DbFit acceptance tests with `gradlew :dbfit-java:netezza:integrationTest`.
```

Comment:
In one of the examples we're calling it via `./gradlew ...`, in the other - `gradlew ...`. That's maybe OK.